### DISCUSSION TITLE
参考 use-antd 示例，如何在 components.view.tsx 里使用 monaco-editor 编辑器组件？

### DISCUSSION THREAD

**Discussion by fankangsong** [opensumi/core#2901] [Q&A]
想脱离文件系统的 URI，在 components.view.tsx 这个页面使用 Monaco-Editor 组件直接展示接口返回的代码；

**Comment by Ricbet** [comment]
@fankangsong 参考一下这里的 editor monaco component 示例 https://github.com/opensumi/opensumi-module-samples/blob/main/modules/editor-monaco-component/README.md
